custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -299,13 +299,21 @@ def _get_mandatory_fields(self, test_case, test_case_properties, occurence_case,
         lab_serial_number = test_case_properties.get('lab_serial_number')
         test_result_grade = test_case_properties.get('result_grade')
         bacilli_count = test_case_properties.get('max_bacilli_count')
+        try:
+            # Since 9 is the max value supported by Nikshay, notify 9 in case 9+
+            if bacilli_count and int(bacilli_count) > 9:
+                bacilli_count = '9'
+        except ValueError:
```

Comment:
Is this exception in case bacilli_count is a string or non-int? what are the expected values for this?

Reply:
@emord it would always be a string only and would mostly be an integer value only. But just added a safe in case its unable to parse the string value as an integer.

Reply:
if it's a non integer value is it safe to keep it as a non integer value? if it's safe, then I approve

Reply:
ya if its a non integer value, it would fail later with `NikshayRequiredValueMissing ` and would get listed as a failed repeat record where we can then track it.